ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -139,13 +139,8 @@ def ocs_install_verification(
     ) == 4.5 and config.ENV_DATA.get('platform') == constants.AZURE_PLATFORM and post_upgrade_verification:
         rgw_count = 1
 
-    # Fetch the min and max Noobaa endpoints from the run config
-    if check_nodes_specs(min_cpu=constants.MIN_NODE_CPU, min_memory=constants.MIN_NODE_MEMORY):
-        min_eps = config.DEPLOYMENT.get('min_noobaa_endpoints')
-        max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints')
-    else:
-        min_eps = 1
-        max_eps = 1 if float(config.ENV_DATA['ocs_version']) < 4.6 else 2
+    min_eps = 1
+    max_eps = 2 if float(config.ENV_DATA['ocs_version']) >= 4.6 else 1
```

Comment:
```suggestion
    max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints') if float(config.ENV_DATA['ocs_version']) >= 4.6 else 1
```

But this is too long so maybe more readable will be 
```suggestion
    max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints') 
    if float(config.ENV_DATA['ocs_version']) < 4.6:
        max_eps = 1
```

Reply:
we should not use config's max_noobaa_endpoints anymore

we don't want to change the CR manually anymore

unless there will be a specific test for it, but this test will change it and not from config

we can remove these keys from default_config